features/frameworks/greentea-client/README.md

```diff
@@ -90,7 +88,7 @@ $ pip install "mbed-greentea<0.2.0" --upgrade
 
 ### utest support
 
-```utest``` is our preferred test harness which allows you to execute a series of (asynchronous) C++ test cases.  ```greentea-client``` includes as test dependency ```utest``` yotta module. See the ```greentea-client``` [module.json](https://github.com/ARMmbed/greentea-client/blob/master/module.json) ```testDependency``` section:
+```utest``` is our preferred test harness which allows you to execute a series of (asynchronous) C++ test cases. See the ```greentea-client``` [module.json](https://github.com/ARMmbed/greentea-client/blob/master/module.json) ```testDependency``` section:
```

Comment:
Let's remove the mention of the `module.json` since it doesn't exist in this repository anymore:
```suggestion
```utest``` is our preferred test harness which allows you to execute a series of (asynchronous) C++ test cases. You can read more about utest [here](https://github.com/ARMmbed/mbed-os/blob/master/features/frameworks/utest/README.md).
```